ex5/ex5.c

```diff
@@ -15,7 +15,26 @@ char* msg3 = "hello world #3";
 
 int main()
 {
-    // Your code here
+    int pipe1[2];
```

Comment:
You actually only need a single `int` array with spots for two ints. A single call to `pipe()` will populate both of those spots with the write handle in the second spot and the read handle in the first spot.